src/composables/use-pool.ts

```diff
@@ -157,11 +173,17 @@ export const usePool = (poolId: number) => {
 
   const fetchExpectedMultiplier = async (lockPeriod: string) => {
     try {
+      if (!lockPeriod) return
+      const lockStart = new Time().isAfter(
+        userPoolData.value?.claimLockStart?.toString(),
+      )
+        ? new Time().timestamp
+        : userPoolData.value?.claimLockStart?.toString()
       const multiplier =
         //eslint-disable-next-line max-len
-        await web3ProvidersStore.erc1967ProxyContract.providerBased.value.getClaimLockPeriodMultiplier(
-          poolId,
-          0,
+        await web3ProvidersStore.erc1967ProxyContract.providerBased.value?.getClaimLockPeriodMultiplier(
```

Comment:
`erc1967ProxyContract` is already extracted to separate variable, no need to invoke `web3ProvidersStore.erc1967ProxyContract` here,

Also in order to reduce long calls like
`erc1967ProxyContract.value.providerBased.value` extract variable up to `providerBased` if other version isn't used in this hook

`const erc1967ProxyContract = computed(() => web3ProvidersStore.erc1967ProxyContract.providerBased)`

Reply:
done